src/ContractData.js

```diff
@@ -76,6 +76,14 @@ class ContractData extends Component {
       displayData = this.context.drizzle.web3.utils.hexToAscii(displayData);
     }
 
+    // If the element has children, let the children render the displayData
+    if (this.props.children) {
+      const childrenWithProps = React.Children.map(this.props.children, child =>
```

Comment:
It looks like you pass the data down to all children? This seems to be a bit much too much to me, it's doing more than is stated. 

I'd recommend a render props approach actually, so that the user can decide what they want to do  with the data.

https://reactjs.org/docs/render-props.html

Reply:
Please see my latest commit. Do I need to do anything else?